pyproject.toml

```diff
@@ -22,17 +22,17 @@ marshmallow = "~3.17.0"
 typing-extensions = "~4.3.0"
 gunicorn = "~20.1.0"
 marshmallow-dataclass = "~8.4"
+jsonschema = "~4.17.0"
 
 [tool.poetry.group.dev.dependencies]
 pylint = "~2.12.2"
-web3 = "~5.28.0"
 psutil = "~5.9.1"
-jsonschema = "~3.2.0"
 pytest-xdist = "~2.5.0"
 pylint-quotes = "~0.2.3"
 black = "~22.6"
 requests = "~2.28"
 isort = "^5.10.1"
+web3 = {version = "6.0.0b4", extras = ["ipfs", "tester"]}
```

Comment:
This is controversial. I bumped this to a new version, because ~5.28.0 would limit us to using jsonschema `<4.0.0` (and also it caused problems when using devnet in starknet.py 😅 )